src/cltk/dependency/processes.py

```diff
@@ -214,3 +217,94 @@ class TreeBuilderProcess(Process):
     def algorithm(self, doc):
         doc.trees = [DependencyTree.to_tree(sentence) for sentence in doc.sentences]
         return doc
+
+
+@dataclass
+class SpacyProcess(Process):
+    """A ``Process`` type to capture everything, that the ``spaCy`` project can do for a given language.
+
+    .. note::
+        ``spacy`` has only partial functionality available for some languages.
+
+    >>> from cltk.languages.example_texts import get_example_text
+    >>> process_stanza = SpacyProcess(language="lat")
+    >>> isinstance(process_stanza, StanzaProcess)
+    True
+    # >>> from stanza.models.common.doc import Document
+    # >>> output_doc = process_stanza.run(Doc(raw=get_example_text("lat")))
+    # >>> isinstance(output_doc.stanza_doc, Document)
+    True
+    """
+
+    language: str = None
+
+    @cachedproperty
+    def algorithm(self):
+        return SpacyWrapper.get_nlp(language=self.language)
+
+    def run(self, input_doc: Doc) -> Doc:
+        output_doc = deepcopy(input_doc)
+        spacy_wrapper = self.algorithm
+        if output_doc.normalized_text:
+            input_text = output_doc.normalized_text
+        else:
+            input_text = output_doc.raw
+        spacy_doc = spacy_wrapper.parse(input_text)
+        cltk_words = self.spacy_to_cltk_word_type(spacy_doc)
+        output_doc.words = cltk_words
+        output_doc.stanza_doc = spacy_doc
+
+        return output_doc
+
+    @staticmethod
+    def spacy_to_cltk_word_type(spacy_doc: spt.Doc):
+        """Take an entire ``spacy`` document, extract
+        each word, and encode it in the way expected by
+        the CLTK's ``Word`` type.
+
+        >>> from cltk.dependency.processes import SpacyProcess
+        >>> from cltk.languages.example_texts import get_example_text
+        >>> process_spacy = SpacyProcess(language="lat")
+        >>> cltk_words = process_spacy.run(Doc(raw=get_example_text("lat"))).words
+        >>> isinstance(cltk_words, list)
+        True
+        >>> isinstance(cltk_words[0], Word)
+        True
+        >>> cltk_words[0]
+
+
+        """
+        def from_spacy_pos_to_upos(pos):
+            for key, value in spacy.parts_of_speech.IDS.items():
+                if pos == value:
+                    return key
+            return ""
+
+        words_list = []  # type: List[Word]
```

Comment:
You can add type annotations in Python, you don't have to resort to comments, and then you would actually get static type checking information.
So instead of writing:
```python
words_list = [] # type: List[Word]
```
You could just write:
```python
words_list: List[Word] = []
```